src/core/linter-rules/no-http-props.js

```diff
@@ -3,18 +3,18 @@
  * Linter rule "no-http-props". Makes sure the there are no URLs that
  * start with http:// in the ReSpec config.
  */
-import { getIntlData, joinAnd, showWarning } from "../utils.js";
+import { docLink, getIntlData, joinAnd, showWarning } from "../utils.js";
 
 const ruleName = "no-http-props";
 export const name = "core/linter-rules/no-http-props";
 
 const localizationStrings = {
   en: {
-    msg: "Insecure URLs are not allowed in `respecConfig`.",
+    msg: docLink`Insecure URLs are not allowed in ${"[`respecConfig`|#configuring-respec]"}.`,
```

Comment:
Maybe we can add `respecConfig` as custom ID in docs instead?

Reply:
I don't mind. Can certainly do later.

Reply:
I edited wiki. Now we've https://respec.org/docs/#respecConfig

Reply:
(we've both actually)